application/tests/harvester_test/change_detector_test.py

```
@@ -0,0 +1,168 @@
+from unittest.mock import patch
+
+from application.utils.harvester.change_detector import (
+    ChangeDetector,
+)
+
+
+class ChangeDetectorTests(unittest.TestCase):
+    @patch("application.utils.harvester.change_detector.subprocess.run")
+    def test_get_modified_files_since(self, mock_run):
+        client = MagicMock()
+        client.get_local_path.return_value = "/tmp/repo"
```

Ruff (0.15.21)

[error] Probable insecure usage of temporary file or directory: "/tmp/repo"

(S108)